ios/PurchasesHybridCommon/PurchasesHybridCommon/CommonFunctionality.swift

```diff
@@ -283,14 +283,19 @@ import RevenueCat
         }
     }
 
-    @objc(purchasePackage:offering:signedDiscountTimestamp:completionBlock:)
+    @objc(purchasePackage:offering:presentedOfferingContext:signedDiscountTimestamp:completionBlock:)
     static func purchase(package packageIdentifier: String,
                          offeringIdentifier: String,
```

Comment:
This would be a breaking change but should we remove the `offeringIdentifier` parameter? FWIW, we decided we would be ok making majors in PHC as often as needed, since it's mostly meant for internal usage.

We can also do this in a separate PR.